src/components/files/FileBrowser.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+	<template v-if="!items.length">
+		<NoFiles />
+	</template>
+	<template v-else>
+		<div class="flex flex-wrap gap-3">
+			<Folder
+				v-for="folder in folders"
+				:modelValue="folder"
+				@update:modelValue="updateItem(folder)"
```

Comment:
Is this not equivalent to the following?

```suggestion
				:key="folder.id"
				v-model="folder"
```